src/main/resources/META-INF/maven/site.vm

```diff
@@ -47,7 +47,8 @@
             <a href="${href}" title="${item.name}">${item.name}</a>
         #end
         <section class="page" itemscope="" itemtype="http://schema.org/BlogPosting">
-            <nav class="menu-section" role="navigation">
+            <h2 style="display: none">${project.organization.name}</h2>
```

Comment:
The html5 specification requires that all sections should contain a header. This section, is the hole page, and the header breaks all styling, while the validation does not consider the visibility of the header, only that it is present in the dom. I can change the tag from section to div, since it makes more sense being a "page" not a section, but my main approach was to maintain the already designed solution.